app/views/dashboards/_subject.html.erb

```diff
@@ -20,18 +20,6 @@
       %></p>
     </div>
   <% end %>
-
-  <% if user.twitter_username.present? %>
-    <div class="flex items-center mb-4 text-b3 lg:text-b2">
-      <%= icon_tag("x-twitter", color: :primary, class: "w-6 text-orange mr-3") %>
```

Comment:
remove the icon here and just use the link in link_to as is in social_link field (if present)